f8a_jobs/graph_ingestion.py

```diff
@@ -10,8 +10,13 @@
 logger = logging.getLogger(__name__)
 
 _INVOKE_API_WORKERS = True \
-    if os.environ.get('INVOKE_API_WORKERS', '1') == '1' \
+    if os.environ.get('INVOKE_API_WORKERS', 'True') == 'True' \
     else False
+
+_DISABLE_UNKNOWN_PACKAGE_FLOW = True \
+    if os.environ.get('DISABLE_UNKNOWN_PACKAGE_FLOW', 'False') == 'True' \
+    else False
```

Comment:
nit:
== returns bool value, no need for if .. else
```suggestion
_DISABLE_UNKNOWN_PACKAGE_FLOW = os.environ.get('DISABLE_UNKNOWN_PACKAGE_FLOW', 'False') == 'True' 
```